usagov/usagov-2021
# usagov-993-datalayer-feedback: update url for Spanish agency index page

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-993

## Description

Update the url from /agencies to /indice-agencias for the 3rd level Spanish datalayer
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Goto http://localhost/es/indice-agencias and "View Page Source" for this page
- In the datalayer information verify the 3rd layer of the url section has "/es/indice-agencias 
- Goto http://localhost/es/agencias/administracion-de-alimentos-y-medicamentos and "View Page Source" and verify the 3rd layer of the url section has "/es/indice-agencias 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
